### PR TITLE
 Add selectAllMatches command in column selection mode.

### PR DESCRIPTION
**Pull request by Inori**
vs_ml
Operation flow:
1.   Ctrl+F                   -->  Open find widegt.
2.   Alt+R                    -->  Turn on regex mode.
3.   Input search text   -->  Regex text or normal text.
4.   **Alt+Enter              -->  Select all matches.**
5.   Left arrow             -->  Ajust cursors.(Ignore this step if you don't want to edit the selected text.)
6.   Edit text                -->  Do what you want.(Ignore this step if you don't want to edit the selected text.)
7.   Shift+Home          -->  Select modified text.(Ignore this step if you don't want to edit the selected text.)
8.   Ctrl+C                   -->  Copy selected text. 
9.   Ctrl+N                   -->  Open a new tab.
10 .Ctrl+V                   -->  Paste.

Compared to sublime:
sb_ml

### CODE REVIEW

**Comment by Inori** [comment]
[ #5447](https://github.com/Microsoft/vscode/issues/5447)

**Comment by alexandrudima** [comment]
@Inori Nice job! :heart:

**Comment by pcarrier** [comment]
Do we know when this will land? It's a killer feature for me and I was disappointed it's not in 1.1.0.

**Comment by taoqf** [comment]
maybe we need 'alt+f3' to select all the same words when some are selected.

**Comment by greatbody** [comment]
just like "ctrl+d" in sublime

File: src/vs/editor/contrib/find/common/findController.ts [hunk 1]
```diff
@@ -220,6 +220,15 @@ export class CommonFindController extends Disposable implements editorCommon.IEd
 		}
 		return false;
 	}
+
+	public selectAllMatches(): boolean {
+		if (this._model) {
+			this._model.selectAllMatches();
+			this.closeFindWidget();
```
Comment (yisibl) [inline src/vs/editor/contrib/find/common/findController.ts]: Why is it removed in the latest code(in master branch)?

**Comment by akinhwan** [comment]
wow did i ever need this, man so useful thank you!!!!!!!

**Pull request merged**